store/src/java/com/zimbra/cs/account/ldap/LdapProvisioning.java

```diff
@@ -1205,7 +1208,9 @@ Account getAccountByName(String emailAddress, boolean loadFromMaster, boolean ch
 
     private Account getAccountByNameInternal(String emailAddress, boolean loadFromMaster)
     throws ServiceException {
-
+        if(null == emailAddress){
```

Comment:
same here

Reply:
I think this is already done ,please find new PR https://github.com/Zimbra/zm-mailbox/pull/1660/commits/025674d8a694f75dddde85bc69db0572742660c9